src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
+
```

Comment:
4. **Unused Parameter:**
   - The `subtract(int a, int b, int c)` method has an unused parameter `c`. Consider removing it as it serves no purpose in the method.